### ISSUE TITLE
Brackets are still colored

### ISSUE THREAD

**Issue by supermitch** [UstymUkhman/matrix-theme#2]
I would prefer shades of green also for brackets.

Currently looks like:
<img width="344" alt="image" src="https://github.com/UstymUkhman/matrix-theme/assets/1152280/bb2ef8f9-d6f7-4bdb-a11f-79782ba99134">

**Comment by UstymUkhman** [comment]
Hi @supermitch! Thanks for your feedback, I will try to fix this if possible as soon as I find some time to come back to this project. In the meantime, feel free to star this theme if you like it to keep me motivated work in it further.

P.S.: For now you can disable default bracket colorization by adding this to your `.code-workspace` file:

```json
"settings": {
  "editor.bracketPairColorization.enabled": false
}
```

**Comment by UstymUkhman** [comment]
Fixed in version `0.1.3`.
